cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/ProgramLifecycleHttpHandlerTest.java

```diff
@@ -1110,8 +1145,7 @@ private void testHistory(Class<?> app, Id.Program program) throws Exception {
       historyStatusWithRetry(getVersionedAPIPath(url, Constants.Gateway.API_VERSION_3_TOKEN, namespace), 2);
 
     } catch (Exception e) {
-      // Log exception before finally block is called
-      LOG.error("Got exception: ", e);
+        LOG.error("Got exception: ", e);
```

Comment:
This was fixed in previous commits : https://github.com/caskdata/cdap/pull/5795/commits/f1fe4835b3d911ab880bf78e0e76fba0dbc147f5
I think @lumig242 reverted the change somehow on the way in later commits. Its fixed in here: 

https://github.com/caskdata/cdap/pull/5837